msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -695,5 +679,23 @@ Define the properties your app uses to post a user activity feed.
 ```
 
 ***
->
->
+
+## defaultInstallScope
+
+**Optional** - string
+
+Specifies whether the scope offers an experience in the context of an individual user alone (personal), a team, a groupchat, or meetings. Options are:
+* `personal`
+* `team`
+* `groupchat`
+* `meetings`
+
+## defaultGroupCapability
+
+**Optional** - object
+
+Specifies the default capability for each collaboration scope your app supports. Options are:
+* `meetings`: `tab`
+* `team`: `bot`
+* `groupchat`: `bot` 
```

Comment:
Could you please change this to table format? 


Name | Type | Maximum size | Required | Description
-- | -- | -- | -- | --
team | string |   |   | When   the install scope selected is Team, this field specifies the default   capability available. Options: tab, bot or connector
groupchat | string |   |   | When   the install scope selected is GroupChat, this field specifies the default   capability available. Options: tab, bot or connector
meetings | string |   |   | When   the install scope selected is Meetings, this field specifies the default   capability available. Options: tab, bot or connector